### PR TITLE
feat: add Ogmios package

### PR DESCRIPTION
**Pull request by verbotenj**
Installing package ogmios (= v6.1.0)
v6.1.0: Pulling from cardanosolutions/ogmios

### CODE REVIEW

File: pkgmgr/package.go [hunk 1]
```diff
@@ -43,6 +43,10 @@ func (p Package) install(cfg Config, context string) (string, error) {
 		cfg.DataDir,
 		pkgName,
 	)
+	pkgConfigDir := filepath.Join(
+		cfg.ConfigDir,
+		pkgName,
+	)
```
Comment (agaffney) [inline pkgmgr/package.go:49]: I don't think we want to use `ConfigDir` for anything other than the actual `cardano-up` config. All package data should stay in the cache/data dirs.

Comment (verbotenj) [reply pkgmgr/package.go:49]: How do I get `config` for "--node-config", "./config/mainnet/config.json"?

Comment (wolf31o2) [reply pkgmgr/package.go:49]: They're already in the container.

Older images were network-specific:
```$ docker run --rm -ti --entrypoint sh cardanosolutions/ogmios:v5.6.0-mainnet -c 'ls -al /config/cardano-node'
total 16
drwxr-xr-x    2 root     root          4096 Aug 12  2023 .
drwxr-xr-x    5 root     root          4096 Aug 12  2023 ..
-rw-r--r--    1 root     root          2725 Aug 12  2023 config.json
-rw-r--r--    1 root     root           128 Aug 12  2023 topology.json
```

While newer images are network agnostic:
```
$ docker run --rm -ti --entrypoint sh cardanosolutions/ogmios:v6.1.0 -c 'ls -al /config/mainnet/cardano-node'
total 16
drwxr-xr-x    2 root     root          4096 Feb 21 10:57 .
drwxr-xr-x    6 root     root          4096 Feb 21 10:57 ..
-rw-r--r--    1 root     root          3006 Feb 21 10:57 config.json
-rw-r--r--    1 root     root           529 Feb 21 10:57 topology.json
```

File: pkgmgr/registry.go [hunk 2]
```diff
@@ -91,6 +91,37 @@ docker run --rm -ti ghcr.io/blinklabs-io/mithril-client:0.7.0-1 $@
 			},
 		},
 	},
+	{
+		Name:        "ogmios",
+		Version:     "v5.6.0-mainnet",
```
Comment (wolf31o2) [inline pkgmgr/registry.go:96]: We'll need 6.0+ for compatibility with our given cardano-node versions.

I suggest `v6.1.0`, instead.

File: pkgmgr/registry.go [hunk 3]
```diff
@@ -91,6 +91,37 @@ docker run --rm -ti ghcr.io/blinklabs-io/mithril-client:0.7.0-1 $@
...
+		Description: "Ogmios, a WebSocket & HTTP server for Cardano, providing a bridge between Cardano nodes and clients.",
+		InstallSteps: []PackageInstallStep{
+			{
+				Docker: &PackageInstallStepDocker{
+					ContainerName: "ogmios",
+					Image:         "cardanosolutions/ogmios:v5.6.0-mainnet",
+					Env: map[string]string{
```
Comment (wolf31o2) [inline pkgmgr/registry.go:103]: Ogmios doesn't read env, but if we wanted to use some and then pipe them through to the command arguments, that could work.

**Review by wolf31o2 — CHANGES_REQUESTED** [review]

**Review by wolf31o2 — APPROVED** [review]

**Pull request merged**